src/services/parseVcf.js

```diff
@@ -35,17 +35,19 @@ export default function parseVcf(data = '', addressbook) {
 
 	importState.total = vCards.length
 
-	return vCards.map(vCard => {
+	// Not using map because we want to only push valid contacts
+	// map force to return at least undefined
```

Comment:
:thumbsup: Good to know and it's nice to see a real life example of reduce being used.

Reply:
:blush: